app/helpers/inventories_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module InventoriesHelper
+  def inventory_shared_status_icon(inventory, team)
+    if inventory.shared_with?(team)
+      if inventory.shared_with_write?(team)
```

Comment:
Can you change this to `can_manage_repository_rows?(inventory)`, because we also need to check that user not a guest.